sf-docker/sf_dependencies/docker-compose.yml

```diff
@@ -9,8 +9,9 @@ services:
       - ACCEPT_EULA=Y
       - SA_PASSWORD=Test1234
     restart: unless-stopped
-    #volumes:
-    #  - /var/lib/smartface/mssql:/var/opt/mssql
+    volumes:
+      - mssqldata:/var/opt/mssql
+      #- /var/lib/smartface/mssql:/var/opt/mssql
```

Comment:
maybe we can now remove the commented lines? or do you see any reason to keep them?